common/src/main/java/com/microsoft/identity/common/CodeMarkersConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.identity.common;
+
+/**
+ * A class having the codemarker value of particular events descriptions.
+ * one or many of these can be used while capturing codemarker event by sending as an artument in method call to markCode of class CodeMarkerManager.
```

Comment:
```suggestion
 * One or many of these can be used while capturing codemarker event by sending as an argument in method call to markCode of class CodeMarkerManager.
```

Reply:
As a person reading this javadoc...I don't know what `markCode` is and I don't know what is `CodeMarkerManager`....I don't want to go around finding these....can we just look to them in javadoc?

Reply:
Hi @shahzaibj, 
I have linked the classes. But markCode method cannot be linked.
Let me know your thoughts please. Thanks

Reply:
Done. Thanks for the suggestion @shahzaibj